server/cmd/mmctl/commands/post.go

```diff
@@ -41,6 +41,22 @@ var PostListCmd = &cobra.Command{
 	RunE: withClient(postListCmdF),
 }
 
+var PostDeleteCmd = &cobra.Command{
+	Use:   "delete [posts]",
+	Short: "Mark posts as deleted or permanently delete posts with the --permanent flag",
+	Long:  `This command will mark the post as deleted and remove it from user's clients but it does not permanently delete the post from the database. Please use the --permanent flag to permanently delete a post and it's attachments from your database.`,
+	Example: `  # Mark Post as deleted
+  $ mmctl post delete udjmt396tjghi8wnsk3a1qs1sw
+
+  # Permanently delete a post and it's file contents from the database and filestore
+  $ mmctl post delete udjmt396tjghi8wnsk3a1qs1sw --permanent true
+
+  # Permanently delete multiple posts and their file contents from the database and filestore
+  $ mmctl post delete udjmt396tjghi8wnsk3a1qs1sw 7jgcjt7tyjyyu83qz81wo84w6o --permanent true`,
```

Comment:
Can we simplify that?
```suggestion
  $ mmctl post delete udjmt396tjghi8wnsk3a1qs1sw --permanent

  # Permanently delete multiple posts and their file contents from the database and filestore
  $ mmctl post delete udjmt396tjghi8wnsk3a1qs1sw 7jgcjt7tyjyyu83qz81wo84w6o --permanent`,
```